docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,575 @@
+
+6. If you previously had the Secure Credential Store plug-in installed, uninstall it now to avoid unexpected behavior. Issue the following command:
+
+    ```
+    zowe plugins uninstall @zowe/secure-credential-store-for-zowe-cli
+    ```
+
+You can now configure the CLI and issue commands.
+## Initializing global configuration
+
+To begin, define a connection to z/OSMF and initialize your configuration files. We recommend this method for getting started, but you can choose create the configuration files manually if desired.
+
+**Follow these steps:**
+
+1. Issue the following command:
+
+   ```
+   zowe config init --global
+   ```
+
+   The CLI provides a series of prompts.
+
+2. Respond to the prompts to enter a service name, username, and password for a mainframe service such as z/OSMF. The `--global` option ensures that your credentials are stored securely on your computer by default.
+
+   After you respond to the prompts, the following file is added to your local `.zowe` directory:
+
+   - `zowe.config.json` - A global configuration file. This is the primary location where your MF service connection details such as host, port, etc... are defined. We recommend that you begin by working with this file.
+
+3. Issue a Zowe CLI command to test that you can access z/OSMF. For example, list all data sets under your user ID:
+
+   ```
+   zowe zos-files list data-set "MY.DATASET.*"
+   ```
+
+   A list of data sets is returned. You successfully configured Zowe CLI to access a z/OSMF instance!
+
+   If the CLI returns an error message, verify that you have access to the target system. Examine your configuration files in a text editor to verify that the information you entered is correct.
+
+**Important!:** After the configuration files are in place (either via the `zowe config init` command or by manually creating the files), the old `zowe profiles` commands will no longer function. Expect to see errors if you attempt to make use of old profiles.
+
+## (Optional) Initializing user-specific configuration
+
+Additionally, you can generate a *user-specific* configuration file. In your user config, you can override values that are defined in the global `zowe.config.json`.
+
+Issue the command `zowe config init --global --user` to generate both global (`zowe.config.json`) and user (`zowe.config.user.json`) config files.
+
+In your user-specific file, notice that the top level defaults, plugins, and secure fields are empty. The profiles do not have any properties. You can add your connection details as properties here to override properties in `zowe.config.json`, or add add new connections.
+## Editing configuration
+
+After the initial setup, you can define additional mainframe services to your global or user config.
+
+Open the `~/.zowe/zowe.config.json` file in a text editor or IDE on your computer. The JSON arrays contain your initial z/OSMF connection details. For example:
```

Comment:
Recommend replacing `The JSON arrays contain your initial z/OSMF connection details` with `The profiles object should contain connection and other frequently needed information for accessing various services.`